.github/workflows/test.yaml

```diff
@@ -32,4 +32,5 @@ jobs:
         run: pip list
       - name: Test with pytest
         run: |
+          export PYTHONPATH=.
```

Comment:
This strikes me as a bit of a code smell. Why do we need it for these tests? The others seemed to be working without it

Reply:
Yes. It has no function here.